lms/djangoapps/verify_student/tests/factories.py

```diff
@@ -6,10 +6,13 @@
 from verify_student.models import SoftwareSecurePhotoVerification
 
 
+# Factories are self documenting
+# pylint: disable=missing-docstring
```

Comment:
pylint is complaining for `missing-docstring` in class `Meta(object):`. For now i will prefer to remain pylint disable statements as is.